.github/workflows/trigger-opslevel-default-branch-name-check.yaml

```diff
@@ -0,0 +1,23 @@
+name: Send default branch name to OpsLevel
+
+on:
+  schedule:
+    - cron: "0 9,12,16 * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: ci-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  trigger-opslevel-default-branch-name-check:
+    name: Trigger Opslevel default branch name check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger check
+        id: opslevel-default-branch-name-check
+        uses: pleo-io/opslevel-default-branch-name-check@main
```

Comment:
Can we use a concrete version for this rather than `main`? 😄

Reply:
Yes, we need to find an agreement about the way how we release the actions.